File: Baekjoon/구현/10872.py
# https://www.acmicpc.net/problem/10872

import sys
input = sys.stdin.readline

def factorial(n):
    if n <= 1:
        return 1
    else:
        return n * factorial(n - 1)
    
n = int(input())
print(factorial(n))

File: Baekjoon/수학/10872.cpp
// https://www.acmicpc.net/problem/10872

#include <iostream>
using namespace std;

int factorial(int n) {
    if (n == 0) {
        return 1;
    }
    return n * factorial(n - 1);
}

int main() {
    int N;
    cin >> N;
    cout << factorial(N) << endl;
    return 0;
}